platforms/common/src/CommonKernels.cpp

```diff
@@ -1664,6 +1665,8 @@ void CommonCalcCustomCentroidBondForceKernel::initialize(const System& system, c
     replacements["COMPUTE_FORCE"] = compute.str();
     replacements["INIT_PARAM_DERIVS"] = initParamDerivs.str();
     replacements["SAVE_PARAM_DERIVS"] = saveParamDerivs.str();
+    replacements["BLOCK_SIZE"] = cc.intToString(this->blockSize);
+    replacements["WARP_SIZE"] = cc.intToString(cc.getSIMDWidth());
```

Comment:
Following on my earlier comments, "warp size" is always defined as 32, regardless of the hardware.  Be aware that `getSIMDWidth()` is not entirely reliable, since there's no standard mechanism for determining it in OpenCL.  If we aren't sure what the width is, that method returns 1.